content/docs/user-guide/components/reference/paintbrush/paintbrush.md

```diff
@@ -0,0 +1,169 @@
+---
+linkTitle: Paint Brush
+title: Paint Brush
+description: ' The Paint Brush is a generalized tool for manipulating arbitrary data in world space using standard painting controls. '
+---
+
+In O3DE, many different components attach data to world space positions and regions. This data can be edited externally in content creation tools, but sometimes it can be easier to edit the data directly in the context of the assembled world inside of O3DE.
+
+The Paint Brush is an editing mode available on some types of components to enable standard painting controls for editing the component's data in world space.
+
+Some components directly map pixels from a source image into world space, but other components might have a more indirect mapping, such as weights stored on vertices on a mesh. The Paint Brush is designed to allow painting in world space for any arbitrary type of data. For that reason, the descriptions below refer more generically to "values" instead of "pixels", even though painting is generally considered to be an image-based operation. Also, the mapping of the data values into world space can be at different densities, so the descriptions below make a distinction between "world space" and "value space". "World space" represents a number of meters in the virtual world. "Value space" represents a number of pixels / vertex weights / etc.
+
+## Using the Paint Brush
+
+The Paint Brush is enabled by clicking the **Edit** button on the component or by selecting the component's icon in the **Component Switcher** in the viewport.
+
+Once the Paint Brush is active, there are three sets of controls:
+
+![Paint Brush Tool image.](/images/user-guide/components/reference/paintbrush/paintbrush-tool.png)
+
+1. A dockable **Paint Brush Settings** window
+2. Paint Brush mode selection icons in the viewport
+3. The Paint Brush circle in the viewport
+
+Painting consists of moving the mouse in the viewport while holding down the left mouse button. A brush stroke begins once the left mouse button is pressed down and ends once the left mouse button is released. Undo/redo commands work on each brush stroke.
+
+A brush stroke consists of a series of overlapping circles with constant spacing that are applied as the mouse moves. The Paint Brush Settings control the color, opacity, size, and spacing of the circles.
```

Comment:
Hm. The sentence is supposed to say that the circles are applied, not that the spacing is applied. How about "A brush stroke consists of a series of overlapping circles applied with constant spacing as the mouse moves." ?

Reply:
We have the description in the previous sentence to rely on too.  Your new sentence works; you could also consider:

> A brush stroke applies a series of overlapping circles with constant spacing as the mouse moves.

or 

> As the mouse moves, a brush stroke applies a series of overlapping circles with constant spacing.

Reply:
Hm. Those are both good suggestions, but they change the meaning a little bit to make it sound like the brush stroke _does_ a thing instead of the brush stroke _is_ a thing. I'll go with my new slightly wordier version.